app/models/ckey2bibframe.rb

```diff
@@ -0,0 +1,42 @@
+# SSH to Symphony and run a catalogdump on a ckey
+class Ckey2bibframe
+  include OperatingSystem
+  include ActiveModel::Model
+  include ActiveModel::Validations
+  attr_accessor :ckey, :baseuri, :marcxml
+  validates :ckey, numericality: { message: ': You must enter a known ckey!' }
+
+  def convert
+    symphony_ssh = "ssh #{Settings.symphony_user}@#{Settings.symphony_host}"
+    symphony_env = "source #{Settings.symphony_env}"
+    catalogdump = "echo #{ckey} | #{Settings.symphony_catalogdump}"
+
+    jar = 'lib/xform-marc21-to-xml-jar-with-dependencies.jar'
+    java_command = "java -Djava.security.egd=file:/dev/../dev/urandom -cp #{jar} edu.stanford.MarcToXMLStream"
+
+    "#{symphony_ssh}" " '#{symphony_env} && #{catalogdump}' | " "#{java_command}"
```

Comment:
The command string must be working, but can't help thinking there are too many quote chars; does it work like:
`  "#{symphony_ssh} '#{symphony_env} && #{catalogdump}' | #{java_command}"  `